provision/vvv-init.sh

```diff
@@ -5,10 +5,10 @@
 
 echo "Beginning WP Meta Environment provisioning"
 
-source ../buddypressorg.test/provision/vvv-init.sh
-source ../jobs.wordpressnet.test/provision/vvv-init.sh
-source ../wordcamp.test/provision/vvv-init.sh
-source ../wordpressorg.test/provision/vvv-init.sh
-source ../wordpresstv.test/provision/vvv-init.sh
+(../buddypressorg.test/provision/vvv-init.sh)
+(../jobs.wordpressnet.test/provision/vvv-init.sh )
+(../wordcamp.test/provision/vvv-init.sh)
+(../wordpressorg.test/provision/vvv-init.sh)
+(../wordpresstv.test/provision/vvv-init.sh)
```

Comment:
What was the reason for this change?

Reply:
when running the first provisioner it would work, but then the second provisioner would be in the wrong folder and they'd all break, sub-shells fixes that though a more recent command changes how they run

Reply:
Ah, ok. I wonder if it'd be more self-documenting to use an absolute path instead?

I prefer `source` over `.` or `()` since it's more more descriptive.

Reply:
Those all mean subtly different things, for example, if `parent.sh` tries to run `child.sh` and `child.sh` has a `cd` then:

 - `source` changes the working directory
 - `()` changes it only for the subshell

`source` executes the script in the current shell, `()` in a subshell. it's a synonym for `.`. I think going forward it's best to use `source` combined with a `cd` after each

Reply:
I'm also not keen on an absolute path as it means you can only ever have 1 meta environment, and if you use the wrong site name it gets broken

Reply:
`source` + `cd` works for me. 

We might also be able so generate absolute paths within the current WME instance with something like this:

https://github.com/WordPress/meta-environment/blob/master/wordcamp.test/provision/vvv-init.sh#L4